src/Util/parseInput.js

```diff
@@ -0,0 +1,10 @@
+const parseInput = userInput =>
+  userInput
+    .trim()
+    .split(',')
+    .map(item => {
+      const [name, quantity] = item.trim().split('-');
+      return [name, Number(quantity)];
+    });
```

Comment:
함수 로직이 여러줄이라 아래처럼 수정하는건 어떨까요??

```suggestion
const parseInput = (userInput) => {
  return userInput
    .trim()
    .split(",")
    .map((item) => {
      const [name, quantity] = item.trim().split("-");
      return [name, Number(quantity)];
    });
};
```

https://github.com/airbnb/javascript#whitespace--implicit-arrow-linebreak
https://github.com/airbnb/javascript#arrows--paren-wrap